lib/ui/sign_transaction/success/sign_transaction_success_page.dart

```diff
@@ -3,37 +3,32 @@ import 'package:get/get.dart';
 import 'package:hypha_wallet/design/background/hypha_half_background.dart';
 import 'package:hypha_wallet/design/background/hypha_page_background.dart';
 import 'package:hypha_wallet/design/bottom_component/hypha_safe_bottom_navigation_bar.dart';
+import 'package:hypha_wallet/design/buttons/button_type.dart';
 import 'package:hypha_wallet/design/buttons/hypha_app_button.dart';
 import 'package:hypha_wallet/design/hypha_colors.dart';
 import 'package:hypha_wallet/design/themes/extensions/theme_extension_provider.dart';
 import 'package:hypha_wallet/ui/bottom_navigation/hypha_bottom_navigation.dart';
+import 'package:hypha_wallet/ui/proposals/details/proposal_details_page.dart';
 
 enum SignSuccessTransactionType {
-  approved('Approved');
+  approved('Approved'),
+  published('published');
 
   const SignSuccessTransactionType(this.value);
 
   final String value;
 
-  Color get iconBackgroundColor {
-    switch (this) {
-      case SignSuccessTransactionType.approved:
-        return HyphaColors.success;
-    }
-  }
+  Color get iconBorderColor => HyphaColors.offWhite;
```

Comment:
Is there a reason you changed the color from HyphaColors.success to HyphaColors.offWhite?

If not, please revert to HyphaColors.success

Reply:
As I mentioned, Luigi updated the UI. The success color was used for the background, but now I'm using off-white for the circle border.